Library/Homebrew/env_config.rb

```diff
@@ -111,6 +111,10 @@ module EnvConfig
         description: "Output this many lines of output on formula `system` failures.",
         default:     15,
       },
+      HOMEBREW_FORBIDDEN_LICENSES:        {
+        description: "Use this environment variable to define a denylist of space separated licenses and Homebrew " \
+                     "will avoid installing the packages with those licenses.",
```

Comment:
Maybe reword this to match the other descriptions on the page (e.g. none of them explicitly refer to "environment variables"). I suggest...

> A space-separated list of licenses. Homebrew will refuse to install a formula if that formula or any of its dependencies has a license on this list.